tests/mocha/connection_checker_test.js

```diff
@@ -341,4 +341,117 @@ suite('Connection checker', function() {
       chai.assert.isFalse(this.checker.doTypeChecks(this.con1, this.con2));
     });
   });
+  suite('Dragging Checks', function() {
+    suite('Stacks', function() {
+      setup(function() {
+        this.workspace = Blockly.inject('blocklyDiv');
+        // Load in three blocks: A and B are connected (next/prev); B is unmovable.
+        Blockly.Xml.domToWorkspace(Blockly.Xml.textToDom(`<xml xmlns="https://developers.google.com/blockly/xml">
+        <block type="text_print" id="wlc#l2D-,6mSLKY=E$VL" x="-76" y="-112">
+          <next>
+            <block type="text_print" id="T9cdR}mwDn3Xl!*g|mw8" movable="false">
+              <comment pinned="true" h="32" w="88.5333251953125">Immovable</comment>
+            </block>
+          </next>
+        </block>
+        <block type="text_print" id="ZgFlirwB{~}%l/BZ+j?R" x="47" y="-118"/>
+      </xml>`), this.workspace);
+      [this.blockA, this.blockB, this.blockC] = this.workspace.getAllBlocks(true);
+      this.checker = this.workspace.connectionChecker;
+      });
+
+      test('Connect a stack', function() {
+        // block C is not connected to block A; both are movable.
+        chai.assert.isTrue(
+          this.checker.doDragChecks(
+            this.blockC.nextConnection, this.blockA.previousConnection, 9000),
```

Comment:
```suggestion
            this.blockC.nextConnection, this.blockA.previousConnection, 9001),
```

:P

Reply:
at some point there will be a test for trying to make a connection when the blocks are too far away, and at that point the distance between the blocks will [make sense](https://en.wikipedia.org/wiki/File:Over_9000!.png)

:P